src/AstAnalyses.cpp

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Are you intending to move all AstAnalyses into AstAnalyses.h/cpp? If not, the filename should be revisited.

Reply:
Ok - let's keep all analyses separate (and separate them in future) as discussed.